services/movie.py

```diff
@@ -1,14 +1,19 @@
 from django.db.models import QuerySet
+from django.db import transaction
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
```

Comment:
If argument can be Nony, you should show it in type hint. Fix everywhere, please
```suggestion
    title: str | None = None
```